src/main/java/teammates/ui/template/StudentInfoTable.java

```diff
@@ -0,0 +1,59 @@
+package teammates.ui.template;
+
+import teammates.common.datatransfer.StudentAttributes;
+import teammates.common.util.Sanitizer;
+
+public class StudentInfoTable {
+    private String name;
+    private String email;
+    private String section;
+    private String team;
+    private String comments;
+    private String course;
+    private boolean isAbleToAddComment;
+    private boolean hasSection;
+
+    public StudentInfoTable(StudentAttributes student, boolean isAbleToAddComment, boolean hasSection) {
+        this.name = student.name;
+        this.email = student.email;
+        this.section = student.section;
+        this.team = student.team;
+        this.comments = student.comments;
+        this.course = student.course;
+        this.isAbleToAddComment = isAbleToAddComment;
+        this.hasSection = hasSection;
+    }
+
+    public String getName() {
+        return Sanitizer.sanitizeForHtml(name);
+    }
```

Comment:
I've been doing sanitization back-end for all my slices.

1. Future developers may be confused on why some things are `c:out`-ed and some things are not.
2. You don't know what's actually happening in `c:out`. It's very clear what `Sanitizer.sanitizeForHtml` does.
3. As mentioned [here](http://tech.finn.no/2011/04/08/xss-protection-whos-responsibility/)